### PR TITLE
Windows 9x MinGW build

### PR DESCRIPTION
**Pull request by glebm**
image

### CODE REVIEW

**Comment by AJenbo** [comment]
Did you try with KernelEx installed?

**Comment by glebm** [comment]
I've installed unicows.exe and KernelEx-4.5.2.exe and now it won't boot :(

**Comment by glebm** [comment]
Now works in 86box with KernelEx!

**Comment by AJenbo** [comment]
Now works on PCem Windows 95 with Pentium II

File: CMake/platforms/mingw9x.toolchain.cmake [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# workaround
+list(APPEND CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES "${CROSS_PREFIX}/i686-w64-mingw32/include")
+
+list(PREPEND CMAKE_C_STANDARD_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_LIST_DIR}/mingw9x/include")
+list(PREPEND CMAKE_CXX_STANDARD_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_LIST_DIR}/mingw9x/include")
+
+# Compatibility with CPUs that do not have MMX.
+set(CMAKE_C_FLAGS_INIT "-mno-mmx")
+set(CMAKE_CXX_FLAGS_INIT "-mno-mmx")
```
Comment (AJenbo) [inline CMake/platforms/mingw9x.toolchain.cmake:15]:
```suggestion
set(CMAKE_C_FLAGS_INIT "-march=pentium")
set(CMAKE_CXX_FLAGS_INIT "-march=pentium")
```

File: CMake/platforms/mingw9x.toolchain.cmake [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# workaround
+list(APPEND CMAKE_CXX_IMPLICIT_INCLUDE_DIRECTORIES "${CROSS_PREFIX}/i686-w64-mingw32/include")
+
+list(PREPEND CMAKE_C_STANDARD_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_LIST_DIR}/mingw9x/include")
+list(PREPEND CMAKE_CXX_STANDARD_INCLUDE_DIRECTORIES "${CMAKE_CURRENT_LIST_DIR}/mingw9x/include")
+
+# Compatibility with CPUs that do not have MMX.
+set(CMAKE_C_FLAGS_INIT "-march=pentium")
+set(CMAKE_CXX_FLAGS_INIT "-march=pentium")
+
```
Comment (AJenbo) [inline CMake/platforms/mingw9x.toolchain.cmake:16]: Lets wait with this par until we get it working.

```suggestion
```

File: Packaging/windows/mingw9x-prep.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+SDLDEV_VERS=1.2.15
+SODIUM_VERS=1.0.18
```
Comment (AJenbo) [inline Packaging/windows/mingw9x-prep.sh:4]: I suppose this is not really needed since NONET is on.

**Review by AJenbo — APPROVED** [review]

**Pull request merged**